tests/5.0/loop/test_loop_reduction_bitor_device.c

```diff
@@ -31,6 +31,9 @@ int test_bitor() {
     have_true = 0;
     have_false = 0;
     for (int x = 0; x < N; ++x) {
+      if( num_attempts == 0 ) {
```

Comment:
Why not initialize a to 0s outside the while ?

Reply:
Where to initialize the array `a` is just an implementation choice, but I agree that initializing the array before the `while` loop will be more efficient; I will change the initialization code as suggested.